readme.md

```diff
@@ -19,21 +19,50 @@ const cryptoRandomString = require('crypto-random-string');
 
 cryptoRandomString(10);
 //=> '2cf05d94db'
+
+cryptoRandomString(10, {type: 'hex'});
+//=> 'c00f094c79'
+
+cryptoRandomString(10, {type: 'base64'});
+//=> 'YMiMbaQl6I'
+
+cryptoRandomString(10, {type: 'url-safe'});
+//=> 'YN-tqc8pOw'
+
+cryptoRandomString(10, {characters: '1234567890'});
+//=> '1791935639'
 ```
 
 
 ## API
 
-### cryptoRandomString(length)
+### cryptoRandomString(length, [options])
 
-Returns a [`hex`](https://en.wikipedia.org/wiki/Hexadecimal) string.
+Returns a randomized string. [`hex`](https://en.wikipedia.org/wiki/Hexadecimal) by default.
 
 #### length
 
 Type: `number`
 
 Length of the returned string.
 
+#### options
+
+Type: `Object`
+
+##### type
+
+Type: `string`
+Values: `hex` `base64` `url-safe`
+
+Setting this option makes it select characters from relevant set. Can not be set at the same time as `characters`
+
+##### characters
+
+Type: `string`
+
+Setting this option makes it select characters from the string. Can not be set at the same time as `type`. Maximum length 65536.
```

Comment:
Should we have a minimum amount of characters? Doesn't really make sense to only have two characters, for example. Can you also mention the entropy aspects of the amount of characters to choose? And a recommended minimum maybe?

Reply:
Don't see any problem with strings length 2. Why not have random string of `0` and `1` for example. Length `0` isn't computable though, so obviously is a limit. Length `1` technically is valid, though isn't random, but to prevent this type of user errors we also need to check unique glyphs, which IMO is a bit of overkill.
> Can you also mention the entropy aspects of the amount of characters to choose?
Don't really understand what are you pointing at. If user doesn't know what entropy is we need to include whole lecture on information theory, and if he knows what entropy is then there is nothing to explain.